notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -0,0 +1,1179 @@
+{
```

Comment:
<p>I would suggest explaining the steps:</p><p>1. Query BQ for all selecting all rows.</p><p>2. Import the rows into a DataFrame</p><p>3. Randomly sort the DataFrame</p><p>4. Export the DataFrame to CSV file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>